flams/run_blast.py

```diff
@@ -69,6 +74,15 @@ def _run_blast(input, modification, lysine_pos, lysine_range, evalue, num_thread
     return [_filter_blast(i, lysine_pos, lysine_range, evalue) for i in blast_records]
 
 
+# BlastpCommandline does not like whitespaces in paths, which the database may contain on especially Mac OS.
```

Comment:
The docstring convention in Python is to put function comments below the header as multiline strings: https://peps.python.org/pep-0257/#multi-line-docstrings